src/utils/is-production.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/**
+ * Returns true when run inside an OpenWhisk action, false otherwise.
+ */
+function production() {
+  // eslint-disable-next-line no-underscore-dangle
+  return !!process.env.__OW_ACTIVATION_ID;
```

Comment:
why not use `NODE_ENV` ?

Reply:
also, what is the purpose of those dumps, if you cannot debug your deployed actions?

Reply:
Are we sure `NODE_ENV` is different in OpenWhisk and Petridish?

Reply:
> also, what is the purpose of those dumps, if you cannot debug your deployed actions?

See https://github.com/adobe/hypermedia-pipeline/pull/61#discussion-diff-222158166R36 the `tap` is only used `when` `!production`, i.e. in Petridish

Reply:
According to this post (http://heidloff.net/article/how-to-create-docker-actions-openwhisk-bluemix) `NODE_ENV` isn't forced to `production` in OpenWhisk, and not reliable.